main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       athlete
+//  action 1:   the athlete runs
+athlete.run();
+//  action 2:   the athlete swims
+athlete.swim();
+//  action 3:   the athlete jumps
+athlete.jump();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       audio effect
+//  action 1:   the audio effect compresses
+audioEffect.compress();
+//  action 2:   the audio effect limits
+audioEffect.limit();
+//  action 3:   the audio effect saturates
+audioEffect.saturate(); 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       face
+//  action 1:   the face frowns
+face.frown();
+//  action 2:   the face laughs
+face.laugh();
+//  action 3:   the face smiles
+face.smile(); 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       eye
+//  action 1:   the eye closes
+eye.close();
+//  action 2:   the eye opens
+eye.open();
+//  action 3:   the eye blinks
+eye.blink(); 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       cat
+//  action 1:   the cat chases a mouse
+cat.chaseMouse();
+//  action 2:   the cat purrs
+cat.purr();
+//  action 3:   the cat pounces
+cat.pounce(); 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       mouse
+//  action 1:   the mouse annoys the cat
+mouse.annoyCat();
+//  action 2:   the mouse eats cheese
+mouse.eatCheese();
+//  action 3:   the mouse runs away from the cat
+mouse.runAwayFromCat();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       finger
+//  action 1:   the finger points
+finger.point();
+//  action 2:   the finger touches
+finger.touch();
+//  action 3:   the finger pokes
+finger.poke(); 

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Reply:
Changes made.